Watchman.Discord/Areas/Administration/Controllers/AdministrationController.cs

```diff
@@ -66,10 +66,7 @@ public async Task ReadUserMessages(DiscordRequest request, Contexts contexts)
                 .OrderBy(x => x.SentAt)
                 .ToList();
 
-            var messagesService = this._messagesServiceFactory.Create(contexts);
-            var hasForceArgument = request.HasArgument("force") || request.HasArgument("f");
-
-            if (messages.Count > 200 && !hasForceArgument)
+            if (messages.Count > 200 && !command.Force)
```

Comment:
fajnie gdybyśmy zminimalizowali ilość na sztywno wpisanych liczb w kodzie

i żeby te informacje były pobierane z konfiguracji -> która aktualnie jest u nas bardzo fajnie zrobiona, więc nie powinno być to dużym problemem

Reply:
ale z jakiej konfiguracji? Jak to pobierać?

Reply:
https://watchman.readthedocs.io/pl/latest/11-how-configuration-works/

spójrz sobie na miejsca w których używamy `IConfigurationService`